tron/core/actionrun.py

```diff
@@ -196,6 +196,27 @@ def to_json(state_data: dict) -> Optional[str]:
             log.exception("Error serializing ActionRunAttempt to JSON:")
             raise
 
+    @staticmethod
+    def from_json(state_data: str):
+        """Deserialize the ActionRunAttempt instance from a JSON string."""
+        try:
+            json_data = json.loads(state_data)
+            deserialized_data = {
+                "command_config": ActionCommandConfig.from_json(json_data["command_config"]),
+                "start_time": (
+                    datetime.datetime.fromisoformat(json_data["start_time"]) if json_data["start_time"] else None
+                ),
+                "end_time": datetime.datetime.fromisoformat(json_data["end_time"]) if json_data["end_time"] else None,
```

Comment:
i'm still a little surprised that we don't need to do any TZ shenanigans here - but given that this seems to be working in infrastage there's probably something weird going on in the pre-existing code that i don't understand :p

Reply:
yea same but it seems to be starting in the right timezone when tz is specified

Reply:
and just to confirm: the start/end times for a completed job with a custom TZ prior to running this branch vs after running this branch display the same in the ui?

(these don't control when the job should start/end, but instead when the job did start/end if i'm not mistaken)

Reply:
yup, we confirmed that through an experiment discussed in slack :p scheduled a job using master branch and then once the job is completed. I started tron with this branch and saw the timings for start time and end time were the same